spring-cloud-kubernetes-core/src/main/java/org/springframework/cloud/kubernetes/KubernetesAutoConfiguration.java

```diff
@@ -33,7 +33,7 @@
 
 @Configuration
 @ConditionalOnProperty(value = "spring.cloud.kubernetes.enabled", matchIfMissing = true)
-@EnableConfigurationProperties(KubernetesClientProperties.class)
+@EnableConfigurationProperties({KubernetesProperties.class, KubernetesClientProperties.class})
```

Comment:
If you're adding it to the additional spring configuration metadata file, there's no need to create a class that isn't used anywhere.